onnxruntime/core/providers/cpu/ml/svmregressor.cc

```diff
@@ -45,31 +45,24 @@ Status SVMRegressor<T>::Compute(OpKernelContext* ctx) const {
 
   Tensor* Y = ctx->Output(0, TensorShape({N, 1}));  // this op outputs for one target only
   const auto* x_data = X->template Data<T>();
+  int64_t current_weight_0;
+  float sum;
```

Comment:
Not sure moving an int64_t and float changes anything performance-wise as I wouldn't have expected any additional initialization cost if these were declared within the loop (conversely something like std::vector is vastly different and worth moving outside).